api/src/settings.py

```diff
@@ -42,7 +42,8 @@ class Settings(BaseSettings):
     PLUGINS: Dict[str, str] = {
         # "my_plugin": "plugin_module.MyPlugin",  # Where plugin_module.MyPlugin is an importable dotted path and MyPlugin
         # is a subclass of utils.PluginInterface
-        # "causemos": "src.plugins.causemos.CausemosPlugin",
+        "causemos": "src.plugins.causemos.CausemosPlugin",
+        "sync": "src.plugins.sync.SyncPlugin",
```

Comment:
I think this gets overwritten by prod's plugin entry (which is missing this new `sync` property). We should do the same suggestion as before- add the plugins+=`sync` entry by default on envfile.sample.

I'm thinking basically what this PR did:

https://github.com/jataware/dojo-stack/pull/74/files#diff-146894aeacf42e1a854d258950e085970851940b8967ebb32cb2fe16c1875393R44

But maybe we go ahead add add both updates as part of this PR (since it is easy to forget to add this on the envfile both on local and prod).

Let me know if it doesn't make sense otherwise 👍🏼